packages/contracts/CHANGELOG.md

```diff
@@ -42,6 +42,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed the `setSignatureValidator` function and `signatureValidator` variable in `DAO`. In places, where the function must remain to not alter the `IDAO` interface ID, it will revert and explanatory notes are put in place.
 - Removed `active_contracts.json` file and replaced it with `@aragon/osx-commons-configs` package.
 
+### Fixed
```

Comment:
👌 
Note for @Rekard0 that we didn't do a "Fixed" section in the past, although this would have been correct according to https://keepachangelog.com/en/1.0.0/ (see top).
Maybe we should do it from now and adapt the earlier changelog. It makes sense to treat fixes differently from features/other changes.